### ISSUE TITLE
[1817] Coal Mine ability failed with `undefined method 'ability'`

### ISSUE THREAD

**Issue by johnplanow** [tobymao/18xx#2151]
* Multiplayer? [https://www.18xx.games/game/13574?action=436](https://www.18xx.games/game/13574?action=436)

**Describe the bug**
Clicking on the coal mine abilities (in the Abilities list above the Undo/Redo/Skip buttons) fails to pop up the ability description or affect the UI in any other way.  The Javascript console has the following error:

```
main.js:2 Uncaught 
a {name: "ability", message: "undefined method `ability' for #<Engine::Step::G1817::SpecialTrack:0xd1ee0>", 
args: Array(1), stack: "ability: undefined method `ability' for #<Engine::…d (https://www.18xx.games/assets/main.js:1:15202)"}
```

**Comment by KenKuhn** [comment]
Here's another example of this. Trying to lay C24 with coal token.

https://www.18xx.games/game/13568?action=773

**Comment by crericha** [comment]
Submitted PR #2153 with the fix. Verified in Ken's game that I was able to use the mine ability with the fix.

**Comment by CaptainObi** [comment]
https://18xx.games/game/13536 similar issue occurring here

**Comment by efwin** [comment]
I had same problem in game 13681 trying to lay tile coming out of Scranton towards Southwest

https://www.18xx.games/game/13681

Thanks

**Comment by jenf** [comment]
fixed by #2153
